src/Symfony/Component/Workflow/TransitionBlockerList.php

```diff
@@ -60,10 +60,8 @@ public function isEmpty(): bool
 
     /**
      * {@inheritdoc}
-     *
-     * @return \ArrayIterator|TransitionBlocker[]
```

Comment:
We should keep this annotation because it contains more information than the return type does.